### PR TITLE
Cloudwatch Logs: Send error down to client

### PR DESCRIPTION
**Pull request by zoltanbedi**
**What this PR does / why we need it**:

It is probably not the fix for https://github.com/grafana/grafana/issues/35448 but I saw that whenever an error happens the run query button never stops spinning. This is a possible fix for that.
 
I saw that by trying to run cloudwatch logs without a query. Maybe it would also make sense to add validation to the frontend as well something like that what we have for the log groups.

### CODE REVIEW

**Review by sarahzinger — APPROVED** [review]
sounds like a good idea to me!

File: pkg/tsdb/cloudwatch/live.go [hunk 1]
```diff
@@ -150,6 +150,11 @@ func (e *cloudWatchExecutor) sendLiveQueriesToChannel(req *backend.QueryDataRequ
 
 	if err := eg.Wait(); err != nil {
 		plog.Error(err.Error())
+		responseChannel <- &backend.QueryDataResponse{
+			Responses: backend.Responses{
+				"A": {Error: err},
```
Comment (andresmgot) [inline pkg/tsdb/cloudwatch/live.go:155]: The refId may not be `A`, I think that a better solution can be to send errors in `startLiveQuery` that has that information available. In any case, living this to someone with more experience in the area. It'd be great if some user from #35448 can verify if this solves the issue.

Comment (zoltanbedi) [reply pkg/tsdb/cloudwatch/live.go:155]: You are right! I'm moving this down.

**Review by andresmgot — APPROVED** [review]
LGTM

**Pull request merged**